com.dynamo.cr/.idea/com.dynamo.cr.iml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We don't usually commit idea files. But in this case it simplifies setup a lot. I decided to add it for now but also updated manual how to set up everything from scratch, so I can remove this.